P2_Area_Dom_Int.sql

```diff
@@ -0,0 +1,26 @@
+-- git test
+SELECT	Area_Unit_Number as AREA_UNIT_NUMBER
+		,REPLACE(REPLACE(Area_Name, 'M�xico', 'Mexico'), ' Area', '') as AREA_NAME
+		,CASE
+			WHEN ca.Area_Name LIKE '%Middle East%'          THEN 'ME/Africa N'
+			WHEN ca.Area_Name LIKE '%Europe E%'             THEN 'Europe E'
+			WHEN ca.Area_Name LIKE '%ica W%'                THEN 'NA W'
+			WHEN ca.Area_Name LIKE '%North America Southw%' THEN 'NA SW'
+			WHEN ca.Area_Name LIKE '%M�xico%'               THEN 'Mexico'
```

Comment:
```suggestion
			WHEN ca.Area_Name LIKE '%M�xico%'              THEN 'Mexico'
```

Reply:
I think it must have did that when I transfered my script from ssms to azure. That is weird! I will make sure to fix that.